packages/block-editor/src/components/link-control/settings-drawer.js

```diff
@@ -28,7 +28,7 @@ function LinkSettingsDrawer( {
 	handleSubmitWithEnter,
 	value,
 	settings,
-	onChange,
+	setNewValue = () => {},
```

Comment:
We could reuse the existing `onChange` prop here but pass in a different callback. No need to add the new `setNewValue` prop.